src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js

```diff
@@ -172,6 +173,8 @@ function EntryCommentItem(props) {
             }
             message={notificationText}
             timestamp={timestamp}
+            // description={text}
+            // descriptionLabel={_ts('entryComments.notifications', 'expandButtonLabel')}
```

Comment:
Why add these comments? Are they to be used in future?

Reply:
yes, I'll provide this work to you and you can do it quickly